model/src/guild/audit_log/event_type.rs

```diff
@@ -1,219 +1,330 @@
-    MessageBulkDelete = 73,
+    MessageBulkDelete,
     /// [Message] was pinned to a [channel].
     ///
     /// [Message]: crate::channel::message::Message
     /// [channel]: crate::channel::Channel
-    MessagePin = 74,
+    MessagePin,
     /// [Message] was unpinned from a [channel].
     ///
     /// [Message]: crate::channel::message::Message
     /// [channel]: crate::channel::Channel
-    MessageUnpin = 75,
+    MessageUnpin,
     /// [Integration] was created.
     ///
     /// [Integration]: super::super::GuildIntegration
-    IntegrationCreate = 80,
+    IntegrationCreate,
     /// [Integration] was updated.
     ///
     /// [Integration]: super::super::GuildIntegration
-    IntegrationUpdate = 81,
+    IntegrationUpdate,
     /// [Integration] was deleted.
     ///
     /// [Integration]: super::super::GuildIntegration
-    IntegrationDelete = 82,
+    IntegrationDelete ,
     /// [Stage instance] was created.
     ///
     /// [Stage instance]: crate::channel::stage_instance::StageInstance
-    StageInstanceCreate = 83,
+    StageInstanceCreate,
     /// [Stage instance] was updated.
     ///
     /// [Stage instance]: crate::channel::stage_instance::StageInstance
-    StageInstanceUpdate = 84,
+    StageInstanceUpdate,
     /// [Stage instance] was deleted.
     ///
     /// [Stage instance]: crate::channel::stage_instance::StageInstance
-    StageInstanceDelete = 85,
+    StageInstanceDelete,
     /// [Sticker] was created.
     ///
     /// [Sticker]: crate::channel::message::sticker::Sticker
-    StickerCreate = 90,
+    StickerCreate,
     /// [Sticker] was updated.
     ///
     /// [Sticker]: crate::channel::message::sticker::Sticker
-    StickerUpdate = 91,
+    StickerUpdate,
     /// [Sticker] was deleted.
     ///
     /// [Sticker]: crate::channel::message::sticker::Sticker
-    StickerDelete = 92,
+    StickerDelete,
     /// [`GuildScheduledEvent`] was created.
     ///
     /// [`GuildScheduledEvent`]: crate::scheduled_event::GuildScheduledEvent
-    GuildScheduledEventCreate = 100,
+    GuildScheduledEventCreate,
     /// [`GuildScheduledEvent`] was updated.
     ///
     /// [`GuildScheduledEvent`]: crate::scheduled_event::GuildScheduledEvent
-    GuildScheduledEventUpdate = 101,
+    GuildScheduledEventUpdate,
     /// [`GuildScheduledEvent`] was deleted.
     ///
     /// [`GuildScheduledEvent`]: crate::scheduled_event::GuildScheduledEvent
-    GuildScheduledEventDelete = 102,
+    GuildScheduledEventDelete,
     /// Thread [channel] was created.
     ///
     /// [channel]: crate::channel::Channel
-    ThreadCreate = 110,
+    ThreadCreate,
     /// Thread [channel] was updated.
     ///
     /// [channel]: crate::channel::Channel
-    ThreadUpdate = 111,
+    ThreadUpdate,
     /// Thread [channel] was deleted.
     ///
     /// [channel]: crate::channel::Channel
-    ThreadDelete = 112,
+    ThreadDelete,
+    /// Unknown type for types not yet known to this library
+    Unknown(u8)
```

Comment:
Let's make this a u16 to be safe for the future.. we're nearly halfway to the limit already, and Discord tends to skip ranges of values in the API.